src/main/scala/io/getclump/Clump.scala

```diff
@@ -31,8 +31,9 @@ sealed trait Clump[+T] {
   def apply(): Future[T] = get.map(_.get)
 
   def getOrElse[B >: T](default: => B): Future[B] = get.map(_.getOrElse(default))
-
-  def list[B](implicit ev: T <:< List[B]): Future[List[B]] = get.map(_.toList.flatten)
+  
+  def flatten[B >: T](implicit cbf: CanBuildFrom[Nothing, Nothing, B]): Future[B] = 
```

Comment:
@williamboxhall The old name `list` doesn't make much sense as the method accepts other collections now, but I din't like `flatten` as well. Do you have suggestions?